<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::TableWidget</class>
 <widget class="QWidget" name="Sekura::TableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbTop">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>new</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/007-plus.png</normaloff>:/images/007-plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbEdit">
        <property name="text">
         <string>edit</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/033-pencil-1.png</normaloff>:/images/033-pencil-1.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbDel">
        <property name="text">
         <string>delete</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/003-close.png</normaloff>:/images/003-close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBottom">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pbSelect">
        <property name="text">
         <string>select</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/002-check.png</normaloff>:/images/002-check.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>147</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>close</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/003-close.png</normaloff>:/images/003-close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="5"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
